src/sampling/fibonacci.jl

```diff
@@ -0,0 +1,123 @@
+"""
+	FibonacciSampling(n)
+
+Generate samples on a sphere using the Fibonacci lattice method.
+The parameter `n` specifies the number of points to generate.
+
+## Example
+
+Sample a sphere with 1000 points:
+
+```julia
+sample(Sphere((0,0,0), 1), FibonacciSampling(100))
+"""
+struct FibonacciSampling <: SamplingMethod
+  size::Int
```

Comment:
```suggestion
  n::Int
```
to be consistent with the docstring and facilitate the lives of readers of the code in the future.